ani-cli

```diff
@@ -348,26 +354,41 @@ while :; do
 		printf "$c_blue[${c_cyan}%s$c_blue] $c_yellow%s$c_reset\n" "s" "select episode"
 	fi
 	printf "$c_blue[${c_cyan}%s$c_blue] $c_magenta%s$c_reset\n" "r" "replay current episode"
+	printf "$c_blue[${c_cyan}%s$c_blue] $c_cyan%s$c_reset\n" "a" "search for another anime"
 	printf "$c_blue[${c_cyan}%s$c_blue] $c_red%s$c_reset\n" "q" "exit"
 	printf "${c_blue}Enter choice:${c_green} "
 	read choice
 	printf "$c_reset"
 	case $choice in
 		n)
 			episode=$((episode + 1))
+			open_episode "$selection_id" "$episode"
 			;;
 		p)
 			episode=$((episode - 1))
+			open_episode "$selection_id" "$episode"
 			;;
 
 		s)	printf "${c_blue}Choose episode $c_cyan[1-%d]$c_reset:$c_green " $last_ep_number
 			read episode
 			printf "$c_reset"
 			[ "$episode" -eq "$episode" ] 2>/dev/null || die "Invalid number entered"
+			open_episode "$selection_id" "$episode"
 			;;
 
 		r)
 			episode=$((episode))
+			open_episode "$selection_id" "$episode"
+			;;
+		a)
+			get_search_query ""
+			search_results=$(search_anime "$query")
+			[ -z "$search_results" ] && die "No search results found"
+			anime_selection "$search_results"
+			episode_selection
+			check_input
+			append_history
+			open_selection
```

Comment:
You could add a `continue` statement here instead of adding `open_episode` for every choice.

Reply:
Fixed in the new commit. I wish I did it like this for the first time.